Source/WebCore/dom/Document.cpp

```diff
@@ -10057,6 +10057,9 @@ void Document::addTopLayerElement(Element& element)
         if (dialogElement && dialogElement->isModal())
             return;
         auto result = m_autoPopoverList.add(*candidatePopover);
+#if PLATFORM(IOS_FAMILY) && ENABLE(TOUCH_EVENTS)
+        invalidateRenderingDependentRegions();
+#endif
```

Comment:
Maybe we can get away with only invalidating if `m_autoPopoverList` was empty before? i.e.

```suggestion
        bool wasEmpty = m_autoPopoverList.isEmpty();
        auto result = m_autoPopoverList.add(*candidatePopover);
#if PLATFORM(IOS_FAMILY) && ENABLE(TOUCH_EVENTS)
        if (wasEmpty)
            invalidateRenderingDependentRegions();
#endif
```